imagersite/imager_images/templates/imager_images/library.html

```diff
@@ -0,0 +1,16 @@
+{% extends 'base.html' %}
+{% load thumbnail %}
+
+{% block content %}
+<p>{{albums.first.user.username}}'s Albums:</p>
+{% for album in albums.all %}
+<p>{{album.title}}<p>
+<p>{{album.description}}</p>
+<p>{{album.date_created}}</p>
```

Comment:
Just like for the photos, you should have a link to the album detail as well.

Reply:
fixed

Reply:
done